lib/hacluster.pm

```diff
@@ -1073,22 +1090,25 @@ sub collect_sbd_delay_parameters {
 
 Calculates start time delay after node is fenced.
 Prevents cluster failure if fenced node restarts too quickly.
```

Comment:
```suggestion
Calculates and return value for start time delay parameter.
This parameter dictate delay time after node is fenced to prevents cluster failure if fenced node restarts too quickly.
```

What does  `used` refer to in sentence: 

```
Delay time is used either if specified in sbd config variable B<SBD_DELAY_START>
or calculated by the formula:
```

Does `used` means "in the code of this funtion"? Or during the cluster lifetime?

Reply:
I'm not really sure. I see the function is calculating a time if it gets `yes` or `1` as an argument, returning 0 if it gets `no` or `0`, returning the actual value it got if it's a number or returning a default value of 35 seconds scaled in case it gets anything else.

Reply:
Applied a reworded version of the suggestion.